tools/logger/logger.c

```diff
@@ -286,7 +286,8 @@ int main(int argc, char *argv[])
 	if (config.version_fw) {
 		config.version_fd = fopen(config.version_file, "rb");
 		if (!config.version_fd) {
-			fprintf(stderr, "error: Unable to open version file %s\n",
+			fprintf(stderr,
+				"error: Unable to open version file %s, check permissions\n",
 				config.version_file);
```

Comment:
I think it is better to add strreror(errno). In my case most common problem is that I forget to add sof debugging support in linux so the debugfs file isn't even there.

Reply:
ack, IIUC the strerror() also has language translations.